en/02_Developer_Guides/09_Security/04_Sudo_Mode.md

```diff
@@ -19,39 +38,24 @@ SilverStripe\Security\SudoMode\SudoModeService:
   lifetime_minutes: 25
 ```
 
-## Enabling sudo mode for controllers
+## Enabling sudo mode for controller endpoints
 
-You can add the `SudoModeServiceInterface` singleton as a dependency to a controller that requires sudo mode for one of its actions:
+Performing a sudo mode verification check in a controller endpoint by using the sudo mode service to validate the request:
 
 ```php
 namespace App\Control;
 
-class MyController extends Controller
-{
-    private ?SudoModeServiceInterface $sudoModeService = null;
-
-    private static array $dependencies = ['SudoModeService' => '%$' . SudoModeServiceInterface::class];
-
-    public function setSudoModeService(SudoModeServiceInterface $sudoModeService): static
-    {
-        $this->sudoModeService = $sudoModeService;
-        return $this;
-    }
-}
-```
```

Comment:
Why is this example being removed?

Reply:
I really dislike the `$dependencies` way of doing it. It's long and unintuitive. I updated the example below to:

```php
$service = Injector::inst()->get(SudoModeServiceInterface::class);
if (!$service->check($request->getSession())) {
// ...
```

Reply:
I personally think we should keep it, and that this change is unrelated - but in the interest of moving forward I'll let this go